packages/components/src/index.html

```diff
@@ -21,6 +21,9 @@
 
 <body>
 
+  
```

Comment:
please make sure to always have the index.html clean and not containing any of your component tests